extensions/amp-story-audio-sticker/0.1/amp-story-audio-sticker.css

```diff
@@ -54,12 +53,36 @@ amp-story-audio-sticker {
   width: 100% !important;
   max-width: 100% !important; /* Set the maximum width of the image to 100% to ensure it fits within the container */
   max-height: 100% !important; /* Set the maximum height of the image to 100% to ensure it fits within the container */
+
+  --story-audio-sticker-outline-color: white !important;
+  --story-audio-sticker-dropshadow-color: white !important;
```

Comment:
In other components (eg: interactive components) the CSS variables can be set by the publishers, which means we don't need to do any extra JS to set them based on an attribute. Someone can write in CSS `amp-story-audio-sticker.my-custom-sticker {--story-audio-sticker-outline-color: red}` and it would work. The only requirement is for these to not be important (so they can be overriden).

Any reasons to not use that behavior?

Reply:
Oh I didn't realize setting CSS variables directly is allowed in Web Story world(I thought the only way is through element attributes). This is nice! Done.

Now we don't need to do the CSS value checks in validator as the developers can check it themselves in dev tool. And it also enables developers to put values other than rgb/rgba values too! Thanks!